wpilibc/src/main/native/include/frc/shuffleboard/ShuffleboardInstance.h

```diff
@@ -18,6 +18,8 @@ namespace detail {
 class ShuffleboardInstance final : public ShuffleboardRoot {
  public:
   explicit ShuffleboardInstance(nt::NetworkTableInstance ntInstance);
+  explicit ShuffleboardInstance(nt::NetworkTableInstance ntInstance,
+                                bool report);
```

Comment:
Instead of duplicating the constructor, why not declare the constructor with a default arg `bool report = false`?

Reply:
Also, you only need explicit for 1 parameter constructors. You don't need it for 2 or more parameter contstructors.

If you do the default arg, you only need the default argument in the header, not the cpp file.